production/helm/loki/values.yaml

```diff
@@ -1504,6 +1504,8 @@ singleBinary:
   persistence:
     # -- Enable StatefulSetAutoDeletePVC feature
     enableStatefulSetAutoDeletePVC: true
+    # -- Enable StatefulSetRecreation for changes to PVC size
```

Comment:
This would be great, if this feature could be described at bit more. E.g what happens if set to true